### ISSUE TITLE
Logo Proposal for Electron music player

### ISSUE THREAD

**Issue by memoryskull** [Sogrey/electron-music-player#1]
Hi @Sogrey 
I'm interested in collaborating with this project by delivering a series of logotype options that fits this project for the best!

If interested let me know so we can get started!

Cheers!

**Comment by Sogrey** [comment]
First of all, thank you very much. It is very nice to have a nice logo. In fact, this project is a `electron` training project, a simple version of local music player, and the online music search and play function may be added later, so the project may not be released online, are you still willing?

**Comment by memoryskull** [comment]
Hi @Sogrey 
Of course I would like to collaborate.
It would be very helpful to know if you want any specific characteristics in the logo?

**Comment by Sogrey** [comment]
Dear @memoryskull

I'm not very good at it, so you're free to design it, and I'm sure I'll like it.

对此，我并不是很擅长，所以你可以自由设计它，我相信我会喜欢它。

**Comment by memoryskull** [comment]
ok perfect @Sogrey

**Comment by memoryskull** [comment]
@Sogrey  I would like to know if they would update the read me information?

**Comment by Sogrey** [comment]
@memoryskull 

Yes, it will be updated to show the logo and some development environment information, usage and help

是的，会更新的，会展示您设计的logo和一些开发环境信息，使用方法和帮助
